android/variables.gradle

```diff
@@ -1,14 +1,14 @@
 ext {
     minSdkVersion = 22
-    compileSdkVersion = 33
-    targetSdkVersion = 33
```

Comment:
Saw that in the migration guide, this bump is necessary, but wonder if we can distribute it for lower version. I guess lets get it released for 34, and then try to do a lower version. Or is `minSdkVersion` for that? 🤔

Reply:
after reading about it - yes, that's how it works. You compile using SDK XYZ while still being backward-compatible with the minSdkVersion. In this case it is Android 5.1 or something. We can bump minSdkVersion to exclude old devices. This can be done via Google Play Console too.

Reply:
Nice 👍